### ISSUE TITLE
Plugin idea : Daybed history

### ISSUE THREAD

**Issue by leplatrem** [spiral-project/daybed#232]
Daybed is very handful for collaborative applications. Rapidly, users might expect to track history of models, records, ...
- RSS renderer
- Views to get history of models and records
- Datetimes of creation, modification, ...
- Diffs
- Restore

http://www.designboom.com/history/66.html :)

**Comment by almet** [comment]
what do you mean by history? Like who did what to where?

**Comment by leplatrem** [comment]
Yes, for example :) But we can also imagine datetime creation, last modification, diffs, restore, ...
